<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Rezervacije</class>
 <widget class="QDialog" name="Rezervacije">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rezervacije</string>
  </property>
  <widget class="QLabel" name="labelSifraRezervacije">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>85</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Sifra rezervacije</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDatumRezervacije">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>datum</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelRedniBrojRezervacije">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>85</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Redni broj</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelKomad">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>85</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Komad</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxKomad">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>320</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditRedniBrojRezervacije">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelBrojSedista">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>85</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Broj sedista</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditBrojSedista">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelDatum">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>85</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Datum</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidgetDatum">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>320</width>
     <height>180</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelImeIPrezimeRezervacije">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ime i prezime</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxIzborAkcije">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Izbor akcije</string>
   </property>
   <widget class="QRadioButton" name="radioButtonUpisi">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>25</y>
      <width>50</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Upisi</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonObrisi">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>50</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Obrisi</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonIzvrsi">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Izvrsi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonIzadji">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Izadji</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSifraRezervacije">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
